app/views/admin/amendments/new.html.erb

```diff
@@ -126,6 +126,27 @@
         </div>
       </div>
     </div>
+
+    <% if @amendment.claim.policy == StudentLoans %>
+      <%= claim_form.fields_for :eligibility, @amendment.claim.eligibility, include_id: false do |eligibility_form| %>
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-one-third">
+            <%= form_group_tag(@amendment, :student_loan_repayment_amount) do %>
+              <%= eligibility_form.label :student_loan_repayment_amount, class: "govuk-label" %>
+              <%= errors_tag @amendment, :student_loan_repayment_amount %>
+            <% end %>
+          </div>
+          <div class="govuk-grid-column-two-thirds">
+            <div class="govuk-form-group">
+              <%= eligibility_form.text_field(
+                    :student_loan_repayment_amount,
+                    value: currency_value_for_number_field(eligibility_form.object.student_loan_repayment_amount),
+                    class: "govuk-input govuk-input--width-10") %>
+            </div>
+          </div>
```

Comment:
How would you feel about using the below so that it is visually consistent with the loan amount question that the claimants see?

I think the narrower input would discourage users from entering big numbers as anything over `1000.00` gets pushed so service operators will hopefully see that they have perhaps entered in an extra 0 when they did not mean to. 

![image](https://user-images.githubusercontent.com/13239597/77073123-a92e4d80-69e6-11ea-8e52-967d3dc1313f.png)

```
<div class="govuk-currency-input">
                <span class="govuk-currency-input__unit">&pound;</span>
                <%= eligibility_form.text_field(
                      :student_loan_repayment_amount,
                      value: currency_value_for_number_field(eligibility_form.object.student_loan_repayment_amount),
                      class: "govuk-input govuk-currency-input__input govuk-input--width-10") %>
              </div>
```

If we do this then the current_input.SCSS will need updating.

```
.govuk-currency-input {
  position: relative;
  &__unit {
    position: absolute;
    bottom: 7px;
    left: 7px;
    @include govuk-font(19, bold);
  }

  &__input.govuk-input {
    padding-left: 20px;
  }
}
```

Reply:
Sorted, thanks Rich.

This is what it looks like now:
![Screenshot 2020-03-19 at 14 27 23](https://user-images.githubusercontent.com/2804149/77077927-d3373e00-69ed-11ea-8e8c-fd0036f665b1.png)